tests/Integration/Database/EloquentUpdateTest.php

```diff
@@ -112,7 +112,7 @@ public function testIncrement()
             'counter' => 0,
         ])->delete();
 
-        TestUpdateModel3::increment('counter');
+        TestUpdateModel3::query()->increment('counter');
```

Comment:
I changed this test because the behaviour has changed now. Calling `increment` on a blank model created from scratch should `return false`. We already don't allow `TestUpdateModel3::update([...]);` either so that aligns properly with the expected